tests/AutoReview/ProjectCodeTest.php

```diff
@@ -208,34 +208,63 @@ public function testThatSrcClassesNotExposeProperties($className)
     }
 
     /**
-     * @param string $className
-     *
      * @dataProvider provideTestClassCases
+     *
+     * @param string $testClassName
      */
-    public function testThatTestClassesAreTraitOrAbstractOrFinal($className)
+    public function testThatTestClassesAreTraitOrAbstractOrFinal($testClassName)
     {
-        $rc = new \ReflectionClass($className);
+        $rc = new \ReflectionClass($testClassName);
 
         static::assertTrue(
             $rc->isTrait() || $rc->isAbstract() || $rc->isFinal(),
-            sprintf('Test class %s should be trait, abstract or final.', $className)
+            sprintf('Test class %s should be trait, abstract or final.', $testClassName)
         );
     }
 
     /**
-     * @param string $className
-     *
      * @dataProvider provideTestClassCases
+     *
+     * @param string $testClassName
      */
-    public function testThatTestClassesAreInternal($className)
+    public function testThatTestClassesAreInternal($testClassName)
     {
-        $rc = new \ReflectionClass($className);
+        $rc = new \ReflectionClass($testClassName);
         $doc = new DocBlock($rc->getDocComment());
 
         static::assertNotEmpty(
             $doc->getAnnotationsOfType('internal'),
-            sprintf('Test class %s should have internal annotation.', $className)
+            sprintf('Test class %s should have internal annotation.', $testClassName)
+        );
+    }
+
+    /**
+     * @dataProvider provideTestClassCases
+     *
+     * @param string $testClassName
+     */
+    public function testThatPublicMethodsAreCorrectlyNamed($testClassName)
+    {
+        $reflectionClass = new \ReflectionClass($testClassName);
+
+        $publicMethods = array_filter(
+            $reflectionClass->getMethods(\ReflectionMethod::IS_PUBLIC),
+            function (\ReflectionMethod $reflectionMethod) use ($reflectionClass) {
```

Comment:
can be `static`

Reply:
@SpacePossum Why don't we have `static_lambda` enabled in the `PhpCsFixer` rule set?

Reply:
it is a risky fixer and when applied causes some of the tests to fail because of the mock framework we use (IIRC), but yeah, I would like it on ;)